vm/src/compile.rs

```diff
@@ -608,6 +608,43 @@ impl Compiler {
         let code = self.pop_code_object();
 
         self.prepare_decorators(decorator_list)?;
+
+        // Prepare type annotations:
+        let mut num_annotations = 0;
+
+        // Return annotation:
+        if let Some(annotation) = returns {
+            // key:
+            self.emit(Instruction::LoadConst {
+                value: bytecode::Constant::String {
+                    value: "returns".to_string(),
```

Comment:
Should be ```return``` not ```returns```?

(This is some horrible python magic that python relies on ```return``` being an impossible argument name because it's a keyword.)

Reply:
Nice catch!